components/server/ee/src/billing/billing-mode.ts

```diff
@@ -108,7 +108,9 @@ export class BillingModesImpl implements BillingModes {
         }
         const cbSubscriptions = await this.subscriptionSvc.getActivePaidSubscription(user.id, now);
```

Comment:
Side note: `getActivePaidSubscription` is wrongly named (because it also returns unpaid / free subscriptions 🙄). Thankfully, this is all going away soon. 👋